terraform/db-backup.tf

```diff
@@ -0,0 +1,16 @@
+data "aws_iam_policy_document" "db-backup" {
+  statement {
+    actions = ["s3:PutObject", "s3:Get*"]
+    resources = [
+      "arn:aws:s3:::sql.pennlabs.org/*",
+      "arn:aws:s3:::sql.pennlabs.org"
+    ]
+  }
```

Comment:
SID is optional, I didn't include an explicit allow since it's the default and if we miss it in the future it could cause more confusion than not including it at all, but if you feel strongly about it, I'm okay adding those in